specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2017-03-01/apimdeployment.json

```diff
@@ -215,6 +215,12 @@
             "schema": {
               "$ref": "#/definitions/ApiManagementServiceResource"
             }
+          },
+          "202": {
+            "description": "Accepted",
+            "schema": {
+              "$ref": "#/definitions/ApiManagementServiceResource"
+            }
```

Comment:
@torresdal can you update the example for `ApiManagementService_CreateOrUpdate` to have 202 response as well?

Reply:
@anuchandy  I added 202 to the examples listed under `x-ms-examples`. All good?